### ISSUE TITLE
`editor.accessibilitySupport` should be migrated to `accessibility.support` or similar

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#180362]
`editor.accesssibilitySupport` is used all over the place at this point. We should migrate to a new setting with a name that correctly reflects the broad scope to which it applies. 

cc @alexdima and @isidorn for input

**Comment by isidorn** [comment]
Even though this setting name is not perfect, I think changing settings is usually a lot of extra work.
So my suggestion is to keep the current name (since it is not too bad).

But if you feel strongly about changing it, I think we can do it as long as we are backwards compatible.

**Comment by meganrogge** [comment]
Okay, that makes sense. We can keep it - especially since VS Code is a code editor, I see how that setting name fits
